src/main/java/strategy/impl/ReturnOperation.java

```diff
@@ -0,0 +1,10 @@
+package strategy.impl;
+
+import strategy.OperationHandler;
+
+public class ReturnOperation implements OperationHandler {
```

Comment:
The class `ReturnOperation` is quite simple and it's not clear how it fits into the overall strategy pattern. Make sure that the purpose of this class is well-defined and that it interacts properly with other strategies within the system.